bitset.go

```diff
@@ -45,3 +45,8 @@ func (b *BitSet) Test(offsets []uint) (bool, error) {
 
 	return true, nil
 }
+
```

Comment:
Regarding NewBitSet above, it looks like it'll produce a BitSet that'll always panic. Also, why the redundant wrapping of an internal pointer?